.travis.yml

```diff
@@ -1,37 +1,60 @@
 language: cpp
+#services:
+#  - docker
+
+arch:
+  - amd64
+  - arm64
 
 os:
   - linux
   - osx
+  - windows
 
 compiler:
   - gcc
 
+matrix:
+  fast_finish: true
+  include:
+    - os: linux
+      arch: amd64
+    - os: linux
+      arch: arm64
+    - os: windows
+    - os: osx
+
 addons:
   apt:
     packages:
     - libgtk-3-dev
     - nasm
+    - build-essential
+    - cmake
+    - libjpeg-dev
 
 before_script:
+#  - docker build --tag openpnp-capture-rpi3-arm64:latest . -f docker/Dockerfile.rpi3_arm64 
   - mkdir build
   - cd build
   - cmake ..
 
 script: make
+# docker run -v $PWD:/openpnp-capture openpnp-capture-rpi3-arm64 make
 
 before_deploy:
   - mkdir deploy
+  - if [ $TRAVIS_OS_NAME = windows ]; then cp libopenpnp-capture.dll deploy; cp linux/tests/openpnp-capture-test deploy/openpnp-capture-test-windows; fi
```

Comment:
Depending on how we build (using Visual Studio, probably), I don't think this filename will be correct.

Reply:
Gotcha, we'll see what it generates after I put the windows bits in.